app/handlers/factoid.js

```diff
@@ -158,6 +168,18 @@ function addFact(factData) {
     }
 }
 
+function approveFactByID(factoidID) {
+    try {
+        let id = parseInt(factoidID)
+        let approveFactStmt = db.prepare('UPDATE factoid SET is_approved = 1 WHERE id = ?')
+        let result = approveFactStmt.run(id)
```

Comment:
Let the handlers above do type-checking and parsing, so you can just simply do:
```suggestion
        let approveFactStmt = db.prepare('UPDATE factoid SET is_approved = 1 WHERE id = ?')
        let result = approveFactStmt.run(factoidID)
```